### DISCUSSION TITLE
springboot/swagger

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#165] [Announcements]
# springboot/swagger

关于 Swagger Swagger 是一个用于生成、描述和调用 RESTful 接口的 Web 服务。 想要理解RESTful架构的话，可以戳链接查看阮一峰老师的博客：https://www.ruanyifeng.com/blog/2011/09/restful.html

https://tobebetterjavaer.com/springboot/swagger.html

**Comment by qinple** [comment]
希望二哥写代码的时候带上导包
